README.md

```diff
@@ -18,7 +18,7 @@ feature or via instrumentation, this project is hopefully for you.
 
 ```bash
 # Apply formatting
-$ ./gradlew format
+$ ./gradlew spotlessApply
```

Comment:
The aim here was to not tie closely to a dependency's task in our workflow but if spotless and this usage are in the other otel projects it seems ok to me.